client/src/game/ui/tools/select.vue

```diff
@@ -45,7 +45,8 @@ export default class SelectTool extends Tool {
         this.selectionHelper.globalCompositeOperation = "source-over";
         gameStore.setSelectionHelperId(this.selectionHelper.uuid);
     }
-    onMouseDown(event: MouseEvent): void {
+
+    onDown(lp: LocalPoint, gp: GlobalPoint, event: MouseEvent | TouchEvent): void {
```

Comment:
Drop the gp argument here and actually do the l2g conversion you're doing now in both onMouseDown and onTouchDown in this function